website/src/components/logo.tsx

```diff
@@ -0,0 +1,47 @@
+import { SVGProps } from 'react';
+
+export const ArmadaIcon = (props: SVGProps<SVGSVGElement>) => (
+  <svg
+    xmlns='http://www.w3.org/2000/svg'
+    fill='currentColor'
+    strokeWidth={0}
+    viewBox='0 0 450 425'
+    role='img'
+    {...props}
+  >
+    <title>Armada icon</title>
+    <g transform='translate(-75 -50)'>
+      <g>
+        <polygon points='299.63 143.31 235.45 221.15 195.86 232.7 300.12 72 404.39 232.7 364.8 221.15 299.63 143.31' />
+        <polygon points='185.95 319.05 121.77 396.88 82.17 408.44 186.44 247.73 290.71 408.44 251.12 396.88 185.95 319.05' />
+        <polygon points='413.07 319.05 348.88 396.88 309.29 408.44 413.56 247.73 517.83 408.44 478.24 396.88 413.07 319.05' />
+      </g>
+      <g className='fill-[#383838] dark:fill-white'>
+        <path d='M299.63,185.12l-46.95,44.74s19.16,37.64,46.95,50.88c27.79-13.24,47-50.88,47-50.88Z' />
+        <path d='M186.44,360.63l-47,44.74s19.17,37.64,47,50.88c27.79-13.24,47-50.88,47-50.88Z' />
+        <path d='M413.56,360.63l-47,44.74s19.17,37.64,47,50.88c27.79-13.24,47-50.88,47-50.88Z' />
+      </g>
+    </g>
+  </svg>
```

Comment:
This seems very wacky to me -- why aren't SVGs split out into their own files? While I know it's not what's happening; inlining images/filetypes is a common technique for obfuscation and my preference would be that we don't do it.